src/www/diag_limiter_info.php

```diff
@@ -68,7 +68,7 @@ function getlimiteractivity() {
       });
   }
   function activitycallback(transport) {
-    jQuery('#limiteractivitydiv').html('<font face="Courier" size="2"><pre style="text-align:left;">' + transport.responseText  + '<\/pre><\/font>');
+    jQuery('#limiteractivitydiv').html('<pre style="text-align:left;">' + transport.responseText  + '<\/pre>');
```

Comment:
The backslash is probably not needed and the pre would make more sense when it is not generated in js if possible

Reply:
I think you're right.